scrapers/bank_of_ghana.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from playwright.sync_api import sync_playwright
+from utils import (
+    download_directory,
+    get_browser,
+    get_file_path,
+    get_md5hash,
+    get_standardized_filename,
+    parse_arguments,
+    validate_and_get_download_link,
+)
+
+BASE_URL = "https://www.bog.gov.gh"
+CIRCULAR_URL = f"{BASE_URL}/downloads/supervision-and-regulation-downloads/regulations-directives/"
+
+
+def bank_of_ghana_scraper():
+    with sync_playwright() as pw:
+        browser = get_browser(pw)
+        page = browser.new_page()
+        page.goto(CIRCULAR_URL)
+
+        session = requests.Session()
+
+        pdf_link = page.locator("//a[contains(@class, 'jet-button')]").all()[0]
+        link = pdf_link.get_attribute("href")
+        link = link.replace(BASE_URL, "")
```

Comment:
Instead of overriding the variable

```suggestion
        link = pdf_link.get_attribute("href").replace(BASE_URL, "")
```